docs/boards/Board - ElinF722.md

```diff
@@ -0,0 +1,31 @@
+# Board - Elin F405
```

Comment:
@elin-neo: I just noticed a bug in this - this should be 'Elin F722'. Can you fix please?

Reply:
Opps. OK. Thanks.